ratelimit/redis.go

```diff
@@ -222,28 +224,28 @@ func (c *clusterLimitRedis) oldest(ctx context.Context, clearText string) (time.
 	res, err := c.ringClient.ZRangeByScoreWithScoresFirst(ctx, key, 0.0, float64(now.UnixNano()), 0, 1)
 
 	if err != nil {
-		finishSpan(true)
+		finishSpan(true, true)
```

Comment:
It is really not clear what "allowed" means in the context of "oldest", that is why I suggest we dont set it in this call path.

Reply:
fixed